batchflow/models/torch/unet.py

```diff
@@ -75,6 +75,7 @@ def default_config(cls):
 
         config['body/decoder/order'] = ['upsampling', 'combine', 'block']
         config['body/decoder/blocks'] += dict(layout='cna cna', kernel_size=3, filters=[512, 256, 128, 64])
+        config['body/decoder/upsample/filters'] = [512, 256, 128, 64]
```

Comment:
Why not 
```suggestion
        config['body/decoder/upsample/filters'] = config['body/decoder/blocks/filters']
```

Reply:
It's shorter, and the number of filters for the default configuration is not going to be changed, so there's no need to tie it to block's filters

Reply:
I now see, that setting upsample/filters should indeed be done as you propose, but in build_config, not here